lms/djangoapps/course_home_api/course_metadata/v1/tests/test_views.py

```diff
@@ -46,6 +46,13 @@ def test_get_authenticated_user(self):
         # 'Course', 'Wiki', 'Progress' tabs
         assert len(response.data.get('tabs', [])) == 4
 
+    def test_get_authenticated_not_enrolled(self):
+        CourseEnrollment.enroll(self.user, self.course.id)
+        CourseEnrollment.unenroll(self.user, self.course.id)
+        response = self.client.get(self.url)
+        assert response.status_code == 200
+        assert not response.data.get('is_enrolled')
```

Comment:
Nit: Are you testing for None or False here? If it's explicitly False, I'd say to change this line to not use `.get` as that defaults to None if the key doesn't exist and might be testing something you didn't intend to do.